armory/data/adversarial/ucf101_mars_perturbation_and_patch_adversarial_112x112.py

```diff
@@ -0,0 +1,248 @@
+"""
+UFC101 actiona recognition adversarial dataset generated using non-universal,
+non-patch perturbation attack
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """
+@article{DBLP:journals/corr/abs-1212-0402,
+  author    = {Khurram Soomro and
+               Amir Roshan Zamir and
+               Mubarak Shah},
+  title     = {{UCF101:} {A} Dataset of 101 Human Actions Classes From Videos in
+               The Wild},
+  journal   = {CoRR},
+  volume    = {abs/1212.0402},
+  year      = {2012},
+  url       = {http://arxiv.org/abs/1212.0402},
+  archivePrefix = {arXiv},
+  eprint    = {1212.0402},
+  timestamp = {Mon, 13 Aug 2018 16:47:45 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/abs-1212-0402},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+_DESCRIPTION = """
+Dataset contains five randomly chosen videos from each class,
+totaling 505 videos. Each video is broken into its component frames.  Therefore,
+the dataset comprises sum_i(N_i) images, where N_i is the number of frames in video
```

Comment:
Can you specify whether these videos were taken from the test split, train split, or some combination?

Reply:
Will do.